lcc_cyclos_base/models/res_partner.py

```diff
@@ -82,21 +93,31 @@ def _cyclos_backend_id(self):
         domain = parsed_uri.netloc
         return "cyclos:%s" % domain
 
-    def _cyclos_backend_data(self):
+    def _cyclos_backend(self):
+        # We only support one backend per type for now
+        backend_data = self.env["res.partner.backend"]
+        if self.lcc_backend_ids.filtered(lambda r: r.type == "cyclos"):
+            backend_data = self.lcc_backend_ids.filtered(lambda r: r.type == "cyclos")[
+                0
+            ]
+        return backend_data
+
+    def _cyclos_backend_json_data(self):
         """Prepare backend data to be sent by credentials requests"""
+        backend_data = self._cyclos_backend()
         backend_id = self._cyclos_backend_id
         if not backend_id:  ## is backend available and configured on odoo
             return []
         data = {
             "type": backend_id,
             "accounts": [],
         }
-        if self.cyclos_id:
+        if backend_data.cyclos_id:
             data["accounts"].append(
                 {
-                    "owner_id": self.cyclos_id,
+                    "owner_id": backend_data.cyclos_id,
                     "url": self.env.user.company_id.cyclos_server_url,
-                    "active": self.cyclos_active,
+                    "active": backend_data.cyclos_active,
```

Comment:
`cyclos_active` doesn't exist in res.partner.backend. To replace with `"active": backend_data.status == 'active'` ?

Reply:
Done